bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Caret.java

```diff
@@ -148,7 +152,8 @@ public Font getFont () {
 	checkWidget();
 	if (font == null) {
 		long hFont = defaultFont ();
-		return Font.win32_new (display, hFont);
+		final Font newFont = Font.win32_new (display, hFont, getZoom());
+		return newFont;
```

Comment:
Why create it (even as `final`) just to return it in the next line? Why not simply:

```suggestion
		return Font.win32_new (display, hFont, getZoom());
```

?